elide-core/src/main/java/com/yahoo/elide/core/exceptions/InvalidPredicateException.java

```diff
@@ -10,6 +10,7 @@
 /**
  * Invalid predicate exception.
  */
+@Deprecated
 public class InvalidPredicateException extends HttpStatusException {
```

Comment:
Should we just have it extend BadRequestException?
```suggestion
public class InvalidPredicateException extends BadRequestException {
```